src/containers/product-details/index.tsx

```diff
@@ -1,57 +1,217 @@
+import { useState } from 'react';
 import clsx from 'clsx';
 import Sticky from '@ui/sticky';
 import Button from '@ui/button';
 import GalleryTab from '@components/product-details/gallery-tab';
 import ProductTitle from '@components/product-details/title';
 import { Item } from '@types';
+import BidTab from '@components/product-details/bid-tab';
+import PlaceBet from '@components/product-details/place-bet';
+import Image from 'next/image';
+import Product from '@components/product';
 
 type Props = {
   className?: string;
   space?: number;
   item: Item;
 };
 
-const ProductDetailsArea = ({ space = 1, className, item }: Props) => (
-  <div className={clsx('product-details-area', space === 1 && 'rn-section-gapTop', className)}>
-    <div className="container">
-      <div className="row g-5">
-        <div className="col-lg-7 col-md-12 col-sm-12">
-          <Sticky>
-            <GalleryTab imageUrl={item.image.url} />
-          </Sticky>
-        </div>
-        <div className="col-lg-5 col-md-12 col-sm-12 mt_md--50 mt_sm--60">
-          <div className="rn-pd-content-area">
-            <ProductTitle title={item.name} likeCount={item.likes} />
-            {/* <span className="bid">
-              Height bid{' '}
-              <span className="price">
+const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
+  const [selectedImg, setSelectedImg] = useState(item.image.url);
+
+  const handleImageChange = (img: string) => {
+    setSelectedImg(img);
+  };
+
+  return (
+    <div className={clsx('product-details-area', space === 1 && 'rn-section-gapTop', className)}>
+      <div className="container">
+        <div className="row g-5">
+          <div
+            className="col-lg-7 col-md-12 col-sm-12"
+            style={{ height: '100vh', position: 'sticky' }}
+          >
+            <Sticky>
+              {/* <div className="row">
```

Comment:
Same with the commented code

Reply:
I let this code commented because it'd be used in the future.
It's the section where you can make the toggle of the images aside the big thumbnail.